### PR TITLE
rx65ncloudkit fixes

### PR DESCRIPTION
**Pull request by akarnil**
partial revert of breaking changes to the sample project with extra fixes to get it building on the latest commits

### CODE REVIEW

File: samples/rx65ncloudkit/basic-sample/src/iotconnect_app.c [hunk 1]
```diff
@@ -133,11 +133,9 @@ static void publish_telemetry() {
 
     const char *str = iotcl_create_serialized_string(msg, false);
     iotcl_telemetry_destroy(msg);
-    if (NULL != str) {
```
Comment (nmatthews-witekio) [inline samples/rx65ncloudkit/basic-sample/src/iotconnect_app.c:136]: Can't believe you need to remove:

if(NULL != str) {
}

is this intentional?

File: samples/rx65ncloudkit/basic-sample/src/main.c [hunk 2]
```diff
@@ -20,20 +19,17 @@
...
-#include "hardware_setup.h"
-
 wifi_ip_configuration_t ip_cfg = {0};
 
 extern bool app_startup(NX_IP *ip_ptr, NX_PACKET_POOL *pool_ptr, NX_DNS *dns_ptr);
 
 /* Define the helper thread for running Azure SDK on ThreadX (THREADX IoT Platform).  */
 #ifndef SAMPLE_HELPER_STACK_SIZE
-#define SAMPLE_HELPER_STACK_SIZE      (7 * 1024)
+#define SAMPLE_HELPER_STACK_SIZE      (6 * 1024)
```
Comment (nmatthews-witekio) [inline samples/rx65ncloudkit/basic-sample/src/main.c]: Made stack size smaller?? Is this intentional

File: scripts/setup-project.sh [hunk 3]
```diff
@@ -170,7 +170,7 @@ create_threadx_projects() {
       ;;
     rx65ncloudkit)
       echo Downloading Azure_RTOS_6...
-      ${FETCH} azrtos.zip https://saleshosted.z13.web.core.windows.net/sdk/AzureRTOS/Azure_RTOS_6.1_RX65N_Cloud_Kit_E2Studio_GNURX_Samples_2022_05_25.zip
+      ${FETCH} azrtos.zip https://saleshosted.z13.web.core.windows.net/sdk/AzureRTOS/Azure_RTOS_6.1_ADU_Renesas_RX65N_Cloud_Kit_e2studio_gnurx_Sample_2022_04_28.zip
```
Comment (nmatthews-witekio) [inline scripts/setup-project.sh:173]: Using an older version?  Is this intentional

**Review by nmatthews-witekio — CHANGES_REQUESTED** [review]
Just have a quick look at comments -- if you can confirm the changes are intentional, then I'll approve.

I think in the future you should really break this into 2 PRs -- or at least 2 commits.

1). just the revert (no other changes).
2). other changes.

File: scripts/setup-project.sh (hunk 3 above)
Comment (akarnil) [reply scripts/setup-project.sh:173]: Yes, older version has extra files (ADU content) which the basic sample does not build without, tested original fetch link but does not compile

File: samples/rx65ncloudkit/basic-sample/src/iotconnect_app.c (hunk 1 above)
Comment (akarnil) [reply samples/rx65ncloudkit/basic-sample/src/iotconnect_app.c:136]: Must have been a bugfix that was reverted, will reimplement

**Review by nmatthews-witekio — APPROVED** [review]

**Comment by nmatthews-witekio** [comment]
Ok, given I can't check it here -- have approved.

File: samples/rx65ncloudkit/basic-sample/src/main.c (hunk 2 above)
Comment (akarnil) [reply samples/rx65ncloudkit/basic-sample/src/main.c]: Unintentional, will revert

**Pull request merged**